interpreter/cling/include/cling/Interpreter/Value.h

```diff
@@ -120,65 +139,67 @@ namespace cling {
     ///   dependencies.
     void AssertOnUnsupportedTypeCast() const;
 
-    size_t GetNumberOfElements() const;
+    bool isPointerType() const { return m_TypeKind == kPtrTy; }
```

Comment:
Can this be renamed, please, as it also returns true for when it's an object? E.g. `isPointerOrObject()`? (I don't think we need the "Type" part?)

Reply:
Ping @vgvassilev

Reply:
I renamed it to `isPointerOrObjectType` for consistency with `isBuiltinType` and others.